docs-v2/integrations/all/mindbody.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Mindbody
+sidebarTitle: Mindbody
+---
+
+API configuration: [`mindbody`](https://nango.dev/providers.yaml)
```

Comment:
The same applies here, `mindbody-api-key`.  Could you also please adjust the docs to match the API key `authmode`. This will allow for addition of the other `authmode`.

Reply:
oh sorry, I initially did OAuth but changed it, will fix rn